frontend/views/components/Modal/Modal.vue

```diff
@@ -1,78 +1,100 @@
 <template lang='pug'>
   div
-    component(:is='content')
+    component(:is='content' ref='content')
     component(:is='subcontent[subcontent.length-1]')
 </template>
 <script>
 import sbp from '~/shared/sbp.js'
-import { OPEN_MODAL, LOAD_MODAL, UNLOAD_MODAL, REPLACE_MODAL, CLOSE_MODAL } from '@utils/events.js'
+import { OPEN_MODAL, REPLACE_MODAL, CLOSE_MODAL } from '@utils/events.js'
 
 export default {
   name: 'Modal',
   data () {
     return {
-      // QUESTION: What's the dif between content and subcontent?
-      // What are the use cases of it?
-      content: null,
-      subcontent: []
+      content: null, // This is the main modal
+      subcontent: [] // This is for collection of modal on top of modals
```

Comment:
It seems like there might be opportunity here for simplification too?

Is it possible to only have an array of `content`? I.e. make it be `content: []` instead of `content: null`, and remove the `subcontent`?

Reply:
I'm not sure how it would simplify, do you want to change the url structure?